### DISCUSSION TITLE
[New Framework] Angular

### DISCUSSION THREAD

**Discussion by rdebeasi** [withastro/roadmap#244] [Proposal]
From @sinedied in [this issue](https://github.com/withastro/astro/issues/346):

>Congrats for this project, this looks very interesting 🙂
>
>Is there any plans down the road to also support Angular in addition to other frameworks?

These conversations have moved from the `astro` repo to the `rfcs` repo, so I thought I'd start a new conversation here. Thanks!

**Comment by delucis** [comment]
Hi @rdebeasi!

There is a actually a community integration for Angular support by @brandonroberts that may fit the bill without this needing to be supported officially: <https://www.npmjs.com/package/astro-angular>
